src/ctia/flows/hooks/event_hooks.clj

```diff
@@ -31,6 +52,20 @@
     (->RedisEventPublisher (lr/server-connection redis-config)
                            channel-name)))
 
+(defn kafka-event-publisher []
+  (let [kafka-topic-config
+        (get-in @properties [:ctia :hook :kafka :topic])]
+
+    (log/warn "Setting up Kafka topic")
+    (try
+      (lk/create-topic)
```

Comment:
You could use `kafka.admin.AdminUtils/listTopics` to check if the topic already exists

Reply:
I do both now, so if another process tries to create the topic after the check it's still caught.

Reply:
I think using the try / create is safer in a potentially concurrent system. I'm not against using the listTopics but I'm not sure it will be much nicer.